Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -1403,15 +1403,15 @@ void JitArm64::srawx(UGeckoInstruction inst)
 
     MOVI2R(WA, 32);
     SUB(WC, WA, WC);
-    LSL(WC, RS, WC);
+    LSL(WC, RS, static_cast<int>(WC));
```

Comment:
Bug. Should use LSLV (as you will probably find out if you rebase on latest master)

Reply:
Done.